src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -679,11 +679,11 @@ bool emberAfOperationalCredentialsClusterAddTrustedRootCertificateCallback(
 
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: commissioner has added a trusted root Cert");
 
-    VerifyOrExit(gFabricBeingCommissioned.SetRootCert(RootCertificate) == CHIP_NO_ERROR, status = EMBER_ZCL_STATUS_FAILURE);
+    VerifyOrExit(gFabricBeingCommissioned.SetRootCert(RootCertificate) == CHIP_NO_ERROR, status = EMBER_ZCL_STATUS_INVALID_FIELD);
```

Comment:
Why this change?

Reply:
The spec calls for sending `INVALID_COMMAND` error and it maps to `EMBER_ZCL_STATUS_INVALID_FIELD`.

Reply:
Ah.  That is ... not the mapping I expected, I guess.  ;)